api/src/devcomp/infrastructure/datasources/learning-content/modules/adresse-ip-publique-et-vous.json

```
@@ -4,7 +4,7 @@
   "title": "L'adresse IP publique : ce qu'elle révèle sur vous !",
   "isBeta": true,
   "details": {
-    "image": "https://images.pix.fr/modulix/placeholder-details.svg",
+    "image": "https://assets.pix.org/modulix/placeholder-details.svg",
```

jusqu'à présent on avait rangé dans le dossier `modules`, il faudrait mettre tout au même endroit dans assets
```suggestion
    "image": "https://assets.pix.org/modules/placeholder-details.svg",
```

---

Modification faite !